lib/src/rules/unreachable_from_main.dart

```diff
@@ -42,8 +43,8 @@ void f() {}
 ''';
 
 class UnreachableFromMain extends LintRule {
-  static const LintCode code = LintCode('unreachable_from_main',
-      'Unreachable top-level member in an executable library.',
+  static const LintCode code = LintCode(
+      'unreachable_from_main', 'Unreachable member in an executable library.',
```

Comment:
Consider adding the name of the member to the message for command-line users.

Reply:
Good call, done!